src/SFA.DAS.AdminService.Web/Controllers/MergeOrganisationsController.cs

```diff
@@ -0,0 +1,386 @@
+        public IActionResult SetSecondaryEpaoEffectiveToDate()
+        {
+            var mergeRequest = _mergeSessionService.GetMergeRequest();
+
+            var viewModel = new SetSecondaryEpaoEffectiveToDateViewModel(mergeRequest.SecondaryEpao, mergeRequest.SecondaryEpaoEffectiveTo);
+
+            return View(viewModel);
+        }
+        
+        [HttpPost("merge-organisations/set-effective-to")]
+        public IActionResult SetSecondaryEpaoEffectiveToDate(SetSecondaryEpaoEffectiveToDateViewModel viewModel)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View(viewModel);
+            }
+
+            var mergeRequest = _mergeSessionService.GetMergeRequest();
+
+            mergeRequest.SetSecondaryEpaoEffectiveToDate(int.Parse(viewModel.Day), int.Parse(viewModel.Month), int.Parse(viewModel.Year));
+
+            _mergeSessionService.UpdateMergeRequest(mergeRequest);
+
+            return RedirectToAction(nameof(MergeOverview));
+        }
+
+        [HttpPost("merge-organisations/overview")]
+        public IActionResult MergeOverview(MergeOverviewViewModel viewModel)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View(viewModel);
+            }
+
+            return RedirectToAction(nameof(ConfirmAndComplete));
+        }
+
+        [HttpGet("merge-organisations/confirm-and-complete")]
+        public IActionResult ConfirmAndComplete()
+        {
+            var mergeRequest = _mergeSessionService.GetMergeRequest();
+
+            var viewModel = new ConfirmAndCompleteViewModel
+            {
+                PrimaryEpaoName = mergeRequest?.PrimaryEpao?.Name,
+                SecondaryEpaoName = mergeRequest?.SecondaryEpao?.Name
+            };
+
+            return View(viewModel);
+        }
+
+        [HttpPost("merge-organisations/confirm-and-complete")]
+        public async Task<IActionResult> ConfirmAndComplete(ConfirmAndCompleteViewModel viewModel)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View(viewModel);
+            }
+
+            var userId = _contextAccessor.HttpContext.User.UserId();
+
+            var mergeRequest = _mergeSessionService.GetMergeRequest();
+
+            var mergeCommand = new MergeOrganisationsCommand
+            {
+                PrimaryEndPointAssessorOrganisationId = mergeRequest.PrimaryEpao.Id,
+                SecondaryEndPointAssessorOrganisationId = mergeRequest.SecondaryEpao.Id,
+                SecondaryStandardsEffectiveTo = mergeRequest.SecondaryEpaoEffectiveTo.Value,
+                ActionedByUser = userId
+            };
+            
+            try
+            {
+                await _apiClient.MergeOrganisations(mergeCommand);
+                
+                mergeRequest.MarkComplete();
+
+                _mergeSessionService.UpdateMergeRequest(mergeRequest);
+            }
+            catch (Exception ex)
```

Comment:
If catching the exception it's worth logging it otherwise don't declare the ex (I think Visual Studio and Sonar will flag that ex is not being used)